README.md

```diff
@@ -12,6 +12,48 @@ This is a Scala/Play frontend to allow Limited Companies to provide their inform
 3. Run the frontend locally using
 `sbt 'run 9718 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
 
+### End-Points
+#### POST /journey
+
+---
+Creates a new journey, storing the journeyConfig against the journeyId
+##### Request:
+
+##### Response:
+Status: **Created(201)**
+Example Response body: 
+
+```
+{“journeyStartUrl” : "/testUrl"}
+```
+
+#### GET /journey/:journeyId
+
+---
+Mocks retrieving all the journey data that is stored against a specific journeyID
```

Comment:
It doesn't mock the retrieval, this endpoint is retrieving the data from the database